en/docs/administer/product-security/logins-and-passwords/working-with-encrypted-passwords.md

```diff
@@ -191,10 +210,14 @@ To change any password which we have encrypted already, follow the below steps:
     * On Linux: `./wso2server.sh`
     * On Windows: `./wso2server.bat`
     
-2.  When you run the startup script, the following message will be prompted before starting the server: `[Enter KeyStore and Private Key Password:]`. When this is prompted, the administrator starting the server must provide the private key and keystore passwords using the command-line to proceed. Note that passwords are hidden from the terminal and log files.
+    When you run the startup script, the following message will be prompted before starting the server: `[Enter KeyStore and Private Key Password:]`.
+
+2.  When prompted, you as the administrator who is starting the server must provide the private key and keystore passwords using the command-line to proceed. 
```

Comment:
the private key password and keystore password ?

Reply:
Fixed in https://github.com/wso2/docs-apim/pull/789/commits/9ba49a0403a2a0d620b949ed2475a8cb46a66b87